ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/capella/block/BlockProcessorCapella.java

```diff
@@ -133,12 +167,98 @@ public void processBlsToExecutionChangesNoValidation(
     }
   }
 
+  // process_withdrawals
+  public void processWithdrawals(
+      final MutableBeaconState genericState, final Optional<ExecutionPayload> maybePayload)
```

Comment:
hmm, this is an interesting case we may want to come back to. Currently we could replay blocks with just the payload header whereas this will require the full payload (which is fine as we currently do always have the payload). At some point we probably want to split this into separate processing and verification so we don't need the full payload.

Reply:
raised #6425 to capture idea!